tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -260,6 +260,7 @@ def get_rocm_test_compilation_infos(
     schedules = []
     if intrinsic == "MFMA":
         schedules = [
+            MMASchedule("MFMA_F32_16x16x4_F32", 1, 1, 1, 1, 1),
```

Comment:
Can we test with more realistic schedules too?

Reply:
Sure.